salt/_roster/kubernetes_nodes.py

```diff
@@ -0,0 +1,55 @@
+import fnmatch
+import logging
+log = logging.getLogger(__name__)
+
+try:
+    import kubernetes.client
+    import kubernetes.config
+    HAS_DEPS = True
+except ImportError:
+    HAS_DEPS = False
+
+__virtualname__ = 'kubernetes'
+
+def __virtual__():
+    if HAS_DEPS:
+        return HAS_DEPS
+    else:
+        return False, 'Unable to load the kubernetes nodes roster: dependencies unavailable.'
+
+
+def targets(tgt, tgt_type='glob', **kwargs):
+    if tgt_type != 'glob':
+        log.error('Only "glob" lookups are supported for now')
+        return {}
+
+    client = None
+    try:
+        client = kubernetes.config.new_client_from_config(
+            config_file='/etc/kubernetes/admin.conf',
```

Comment:
Please file a debt ticket to change the `kubeconfig` file being used.

Reply:
#834